src/components/Shared/Filters/helpers.ts

```diff
@@ -41,4 +41,15 @@ const reverseFiltersOptions = {
   user_ids: 'reverseUserIds',
 };
 
+export const renameFilterNames = (name: string | undefined) => {
```

Comment:
I wonder if this should be done at the API-level instead? At least eventually. @dbenton9

Reply:
What's happening here @canac?

Reply:
Sorry for the lack of context. We need to rename "referrals" to "connections" everywhere in MPDX. Some of the places where "referrals" are showing up is in filter options. I think that eventually we'd want to rename the filter titles in the API. But not without some coordination first because that change will affect Angular and presumably the mobile apps. So I'm not asking for any action on your part right now just mentioning that we may need to in the future.

Reply:
@dbenton9 It is not urgent but it looks like it would be a simple fix since each `filter` and `group` have a `title` that can be changed.
`  accountList(id: $accountListId) {
    contactFilterGroups {
      filters {
        filterKey
        title
        ... on MultiselectFilter {
          title
          options {
            
          }
        }`